### PR TITLE
UI : fix request/update task after redirect from frequently joined tables

### LINKED ISSUE #17448: UI Bug on Database Level when applied Restricted View on Rules & Policies
**Affected module**
UI affected, specifically on Database Level page

**Describe the bug**
Compared to the Database Schema, Tables, and Database Service pages, when the Database level page is restricted, it doesn't display a "You don't have access" message. Instead, it shows an infinite loading screen.

**To Reproduce**

Apply rules and policies that prevent users from viewing any data assets that do not belong to them (i.e., where they are not the owner).
screenshot_2024-08-14_at_14 49 11

Then try to open the Database Level data asset. It'll show an infinite loading screen, like this.
screenshot_2024-08-14_at_14 32 02

To be compared, here the screenshot when I'am open other level of data asset.
screenshot_2024-08-14_at_14 31 25

**Expected behavior**
The expected, is just like the other, when I'am open the Database Level data asset, it will show an "You don't have access" message

**Version:**
 - OS: macOS 14.0
 - Python version: ?? using the omd dependencies for the airflow
 - OpenMetadata version: 1.4.4
 - OpenMetadata Ingestion package version: 1.4.4 following the omd version

**Additional context**
https://openmetadata.slack.com/archives/C02B6955S4S/p1723620621805789

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

- Fix Task page breaking for table on request/update task after redirecting from frequently joined tables
- Fix the API breaking of conversation in task for tags/description after redirecting from frequently joined tables
- Fix the truncating of tags
- Change persona header to entity like headers with icon
- 
- Fix #17448 

Issue : 

https://github.com/user-attachments/assets/d4a2d381-b457-4259-ba62-773d02097028

image

Resolved : 

https://github.com/user-attachments/assets/a0d8e2da-c1a3-4f8a-b90f-5856d9fc0990

image
image

#
### Type of change:

- [x] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Tag/TagsV1/TagsV1.component.tsx [hunk 1]
```diff
@@ -153,7 +153,7 @@ const TagsV1 = ({
         {...tagProps}>
         {/* Wrap only content to avoid redirect on closeable icons  */}
         <Link
-          className="no-underline h-full w-full"
+          className="no-underline h-full w-fill-available"
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Tag/TagsV1/TagsV1.component.tsx:156]: Can you check we have enough CSS to support cross-browser?
https://caniuse.com/?search=fill-available

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestDescriptionPage/RequestDescriptionPage.tsx [hunk 2]
```diff
@@ -116,7 +122,7 @@ const RequestDescription = () => {
       const data: CreateThread = {
         from: currentUser?.name as string,
         message: value.title || taskMessage,
-        about: getEntityFeedLink(entityType, decodedEntityFQN, getTaskAbout()),
+        about: getEntityFeedLink(entityType, entityFQN, getTaskAbout()),
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestDescriptionPage/RequestDescriptionPage.tsx:125]: What's rsn for changing decodedFQN with FQN

File: openmetadata-ui/src/main/resources/ui/src/styles/components/size.less [hunk 3]
```diff
@@ -149,6 +149,9 @@
 .w-fit-content {
   width: fit-content;
 }
+.w-fill-available {
+  width: -webkit-fill-available;
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/styles/components/size.less:153]: moz is missing won't work for cross browser

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestDescriptionPage/RequestDescriptionPage.tsx (hunk 2 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestDescriptionPage/RequestDescriptionPage.tsx:125]: It's the decoded fqn itself but in the case of `entityType: Table` it's being modified with `getTaskEntityFQN` and the return value is saved in `entityFQN`

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.ts [hunk 4]
```diff
@@ -504,7 +504,7 @@ describe('Glossary page should work properly', { tags: 'Governance' }, () => {
         interceptURL('GET', `/api/v1/teams/**`, 'getTeams');
         interceptURL(
           'GET',
-          `/api/v1/users?fields=teams%2Croles&limit=25&team=${appName}`,
+          `/api/v1/users?fields=roles&limit=25&team=${appName}`,
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.ts:507]:
```suggestion
          `/api/v1/users?fields=*team=${appName}`,
```

**Review by chirag-madlani — DISMISSED** [review]

File: openmetadata-ui/src/main/resources/ui/src/pages/Persona/PersonaDetailsPage/PersonaDetailsPage.tsx [hunk 5]
```diff
@@ -65,7 +68,7 @@ export const PersonaDetailsPage = () => {
         ),
       },
       {
-        name: getEntityName(personaDetails),
+        name: personaDetails?.name ?? '',
         url: '',
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/pages/Persona/PersonaDetailsPage/PersonaDetailsPage.tsx:72]: Why do we need this change?

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**